saltproc/fuel_comp_with_fix

```diff
@@ -0,0 +1,1159 @@
+mat  fuel  7.77767011499957E-02 fix 10c 1000 burn 1 vol 4.87100E+07 % density 3.266364g/cm3 for 1000K
```

Comment:
I feel this shouldn't be added to the top level directory. Perhaps there is a more organized place to place it.